install/operator/pkg/apis/syndesis/v1beta1/syndesis_types.go

```diff
@@ -157,11 +157,10 @@ type VolumeAccessMode string
 const (
 	ReadWriteOnce VolumeAccessMode = "ReadWriteOnce"
 	ReadOnlyMany  VolumeAccessMode = "ReadOnlyMany"
-	ReadWriteMany VolumeAccessMode = "ReadWriteMany"
```

Comment:
This is a possible Volume Access Mode so why is it being removed?
eg. https://docs.openshift.com/container-platform/4.2/storage/understanding-persistent-storage.html

Reply:
Because it is not used anywhere and my go-linter doesn't like it. We can comment it out maybe?